glue/sample/src/sinter/_decoding_test.py

```diff
@@ -361,7 +361,7 @@ def test_decode_fails_correctly(decoder: str, force_streaming: Optional[bool]):
         with open(d / 'bad_dets.b8', 'wb') as f:
             f.write(b'!')
 
-        if 'vacuous' not in decoder:
+        if 'vacuous' not in decoder and decoder_obj is not None:
```

Comment:
I think this should assert it's not none instead of disabling the testing of the decoder that wasn't.